drivers/entropy/Kconfig.sam0

```diff
@@ -0,0 +1,14 @@
+# Kconfig.sam - Atmel SAM0 TRNG configuration
+#
+# Copyright (c) 2019 Benjamin Valentin
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config ENTROPY_SAM0_RNG
+	bool "Atmel SAM0 MCU Family True Random Number Generator (TRNG) Driver"
+	depends on SOC_FAMILY_SAM0
+	default y
```

Comment:
why default y?

Reply:
So it gets selected automatically if `ENTROPY_GENERATOR` is selected and the sam0 family is used.
I just copied that from the other drivers - but actually it should be

```
depends on SOC_FAMILY_SAMD5X
```

Reply:
I understand that, but it seems to me that this is not the case for several other entropy drivers.

Reply:
I will drop it then